core/src/components/header/header.ios.scss

```diff
@@ -56,10 +56,14 @@
  */
 .header-collapse-condense ion-toolbar {
   --background: var(--ion-background-color, #fff);
-  
+
   z-index: 0;
 }
 
+.header-collapse-condense ion-toolbar:last-of-type {
```

Comment:
TODO: Would it be better to just make the initial selector not apply the collapsible large title instead? https://github.com/ionic-team/ionic-framework/blob/faf36f9c0795f67dbd40586c25a1eec5c424c094/core/src/components/header/header.ios.scss#L7 Maybe easier to override?

Reply:
Follow up: not doing this because it would make it harder to override for non-large title toolbars.